nuitka/plugins/standard/PySidePyQtPlugin.py

```diff
@@ -137,6 +137,7 @@ def considerExtraDlls(self, dist_dir, module):
                 # Seems platforms is required on Windows.
                 if os.name == "nt":
                     plugin_options.add("platforms")
+                    plugin_options.add("styles") # Makes Qt window look much better, than before
```

Comment:
So this is limited to Windows (os.name == "nt") then? Would it apply to other platforms too.

Reply:
This folder contains qwindowsvistastyle.dll, so I think yes